piqueserver/extensions.py

```diff
@@ -0,0 +1,101 @@
+
+import os
+import importlib
+import imp
+
+def check_scripts(scripts):
+    '''
+    Ensures that there are no duplicate scripts in scripts
+    :param scripts: list of scripts
+    :return: True if there is no duplicate scripts, False otherwise
```

Comment:
we use napoleon docstring syntax: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/

Reply:
To be more specific, we use the google style yes, since you asked in your wiki. You shouldn't add types there though, you can just leave the `(int)` off. If you really want to add types, you should [add them to the function signature instead](https://www.python.org/dev/peps/pep-0484/) to help with autocomplete. That's not very important though.

Reply:
Ok, very helpful. We'll try to produce documentation according to that style then.
Thanks